tests/test_db_function.py

```diff
@@ -53,3 +58,43 @@ def test_non_blob_columns_mutable(db_session):
         db_session.commit()
     except sqlalchemy.exc.InternalError:
         raise AssertionError
+
+
+def test_create_scores_for_application_su_crit():
+    picked_row = get_random_row(AssessmentRecord)
+    application_id = picked_row.application_id
+    sub_criteria_id = "app-info"
+    
+    assessment_payload = {
+        "application_id": application_id,
+        "sub_criteria_id": sub_criteria_id,
+        "score": 3,
+        "justification": "bang average",
+        "user_id": "test"
+    }
+    score_metadata = create_just_score_for_application_sub_crit(**assessment_payload)
+
+    assert len(score_metadata) == 7
+    assert score_metadata.get("date_created")
+    assert score_metadata.get("score") == 3
```

Comment:
⛏️ Don't use `get` when an object must have this property. It allows for `None`s to propagate though the code.

Reply:
another great shout!